include/swift/ABI/TrailingObjects.h

```diff
@@ -97,27 +97,20 @@ class TrailingObjectsBase {
   template <typename T> struct OverloadToken {};
 };
 
-/// This helper template works-around MSVC 2013's lack of useful
-/// alignas() support. The argument to LLVM_ALIGNAS(), in MSVC, is
-/// required to be a literal integer. But, you *can* use template
-/// specialization to select between a bunch of different LLVM_ALIGNAS
-/// expressions...
```

Comment:
If this comment is no longer relevant, probably all of TrailingObjectsAligner isn't either. What's LLVM doing here these days?

Reply:
Little divergence: http://llvm.org/r338099
Though they kept the comment! I'll remove it from upstream...

Reply:
llvm.org/r367336